features/bevragen/zoek-met-postcode-en-huisnummer/fout-cases.feature

```diff
@@ -149,7 +62,7 @@ Rule: Postcode en huisnummer zijn verplichte parameters
     | required | huisnummer | Parameter is verplicht. |
 
   @fout-case
-  Scenario: Zoek met leeg postcode en leeg huisnummer 
+  Scenario: Een lege string is opgegeven als postcode en huisnummer waarde 
```

Comment:
je test alleen met lege string. vanuit testoogpunt mis ik dan sturen van null (niet: "null") en niet sturen van huisnummer (parameter huisnummer ontbreekt in de zoek body)

Reply:
> je test alleen met lege string. vanuit testoogpunt mis ik dan sturen van null (niet: "null") en niet sturen van huisnummer (parameter huisnummer ontbreekt in de zoek body). De Gba API geeft nu bij sturen huisnummer: null een andere foutmelding dan bij ontbreken van huisnummer.

Geef dan aan mbv een scenario hoe je dat wilt beschrijven. Ik ga niet iets implementeren en dan later kritiek krijgen dat het niet is wat jij wilt

Reply:
laten we zo